### PR TITLE
No GSL

### PR DESCRIPTION
**Pull request by HaoZeke**
We should also rebase once this is in.

### CODE REVIEW

**Comment by amritagos** [comment]
Merging for now, but we should transition the build system as well.

File: src/include/internal/generic.hpp [hunk 1]
```diff
@@ -18,7 +18,9 @@
 
 // Boost
 #include <boost/math/constants/constants.hpp>
-#include <gsl/gsl_blas.h>
+// Eigen
+#include <eigen3/Eigen/Core>
+#include <eigen3/Eigen/Dense>
```
Comment (amritagos) [inline src/include/internal/generic.hpp:23]: Ideally these shouldn't be included in multiple places.

**Review by amritagos — APPROVED** [review]
LGTM! Thanks @HaoZeke.

File: src/include/internal/generic.hpp (hunk 1 above)
Comment (HaoZeke) [reply src/include/internal/generic.hpp:23]: Yup, maybe in a followup.

**Pull request merged**